src/task/task.controller.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  HttpCode,
+  Param,
+  ParseIntPipe,
+  Put,
+  Query,
+} from '@nestjs/common';
+import { TaskService } from './task.service';
+import { Task } from 'src/entities/task.entity';
+import { ApiQuery } from '@nestjs/swagger';
+import { UpdateTaskDto } from './dto/update-task.dto';
+import { GetUser } from 'src/decorator/get-user.decorator';
+import { User } from 'src/entities/user.entity';
+
+@Controller('task')
+export class TaskController {
+  constructor(private readonly taskService: TaskService) {}
+
+  @Get()
+  @ApiQuery({ name: 'workspaceId' })
+  @ApiQuery({ name: 'userId' })
+  @ApiQuery({ name: 'isForCurrentUserOnly' })
+  async getDefaultTaskList(
+    @Query('workspaceId') workspaceId: number,
+    @Query('isForCurrentUserOnly') isForCurrentUserOnly: boolean,
+  ): Promise<{ tasks: Task[] }> {
+    return await this.taskService.getDefaultTaskList(
+      workspaceId,
+      1,
```

Comment:
Why 1 is hardcoded? We don't need this id, instead we should rely on GetUser decorator